### PR TITLE
Add ensa version sap system details

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Add ensa version value to SAP system details view. **By now, using `main` branch the default "-" will be shown, as the backend part is `deregistration` branch. I'm sending the PR to `main` to avoid yet much more conflicts**

image

Storybook stories added for SAP system and Database details.

## How was this tested?

Tested and stories added

### CODE REVIEW

File: assets/js/components/SapSystemDetails/GenericSystemDetails.jsx [hunk 1]
```diff
@@ -12,6 +12,9 @@ import {
   systemInstancesTableConfiguration,
 } from './tableConfigs';
 
+export const renderEnsaVersion = (ensaVersion) =>
```
Comment (arbulu89) [inline assets/js/components/SapSystemDetails/GenericSystemDetails.jsx:15]: I'm not sure if the place for this function is the best one.
Suggestions?

Comment (dottorblaster) [reply assets/js/components/SapSystemDetails/GenericSystemDetails.jsx:15]: You can both leave it here or have a dedicated file, if we maybe get to have more and more render functions we can have something like `renderers.js` or something like that. For now I think we can leave it

**Review by jagabomb — APPROVED** [review]
LGTM

**Review by dottorblaster — APPROVED** [review]
LGTM

**Review by nelsonkopliku — APPROVED** [review]
Great job!

Well, I feel guilty about the many `keysToCamel` you were forced to add :see_no_evil: 

The initial intent about it was enabling the possibility to transform snake_case api responses/requests to camelCase objects in the code and vice-versa, but it never happened :sweat_smile: 

I guess we could just get rid of it, we use snake_case mixed with camelCase anyway.
Feel also free to track debt.

**Comment by arbulu89** [comment]
> Feel also free to track debt.
Done! Ticket to track this added to our refinement backlog

**Pull request merged**